src/sections/Home/Banner-4/index.js

```diff
@@ -62,13 +62,15 @@ const Banner1 = () => {
               </span>
             </Container>
           </Col>
-          <Col sm={4} lg={6} className="section-title-wrapper video-col">
-            <Link to="/cloud-native-management/meshmap">
-              <video autoPlay muted loop preload="metadata" className="meshmapVideo">
-                <source src={meshmapVideo} type="video/mp4"></source>
-              </video>
-            </Link>
-          </Col>
+          {typeof window != "undefined" && window.innerWidth > 760 && (
```

Comment:
User-agent just wasn't good enough for you, huh? Getting fancy with "typeof" instead. 

All kidding aside: I suppose the combination of these properties is a broader guarantee than trying to match all-known mobile user-agents.

Reply:
Hehe, I had to 😎